web/src/main/java/org/springframework/security/web/servletapi/HttpServlet3RequestFactory.java

```diff
@@ -32,6 +32,7 @@
 import org.apache.commons.logging.LogFactory;
 
 import org.springframework.security.authentication.AuthenticationCredentialsNotFoundException;
+import org.springframework.security.authentication.AuthenticationDetailsSource;
```

Comment:
Will you please update the copyright message to `2021` for classes that you modify?